estate/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "estate app",
+    "summary": "Estate module",
+    "description": "Estate module",
+    "author": "Odoo",
+    "category": "Tutorials/estate",
+    "version": "0.1",
```

Comment:
```suggestion
    "version": "1.0",
```
Usually we start versioning from 1.0 because main version should start from 1